pkgs/tools/misc/piston-cli/default.nix

```diff
@@ -1,15 +1,15 @@
-{ lib, python3Packages }:
+{ lib, python39Packages }:
```

Comment:
Please overwrite the passed in python3Packages.

Reply:
I don't understand what you mean?

Reply:
In `pkgs/top-level/all-packages.nix`:
```nix
  piston-cli = callPackage ../tools/misc/piston-cli { python3Packages = python39Packages; };
```